boot/makebootable.go

```diff
@@ -330,6 +329,21 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		return fmt.Errorf("cannot set run system environment: %v", err)
 	}
 
+	_, ok = bl.(bootloader.ManagedAssetsBootloader)
+	if ok {
+		// the bootloader can manage its boot config
+
+		// XXX: this must be performed after the boot partition has been
```

Comment:
this is not really a XXX, the order it's not something intrinsically bad  (the double writing is a bit but that's covered elsewhere), so s/XXX/NOTE/

Reply:
Done